site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -16,18 +16,15 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
 ## Table of Contents
 
 - [Background](#background)
-- ["Hover" triggering](#hover-triggering)
-  - [Add the `interest` value](#add-the-interest-value)
-  - [CSS values to control delay](#css-values-to-control-delay)
-  - [Example buttons](#example-buttons)
-  - [Popover types](#popover-types)
-  - [Example menu](#example-menu)
-- [`popover=hint` Behavior](#popoverhint-behavior)
+  - [What is a tooltip/toggletip](#what-is-a-tooltiptoggletip)
+  - [Tooltip lifecycle behaviors](#tooltip-lifecycle-behaviors)
+- [This Explainer is only about #2](#this-explainer-is-only-about-2)
```

Comment:
This title seems odd. Can you adjust it to say something like "Scope of this explainer" and then note that 2 & 3 are out of scope for this document.

Reply:
Done.